native/src/seal/polyarray.h

```diff
@@ -45,6 +45,25 @@ namespace seal
                     const SEALContext &context, const PublicKey &public_key, MemoryPoolHandle pool
                 );
 
+                /**
```

Comment:
Thinking about it now, you should delete the default constructor by doing

```cpp
PolynomialArray() = delete;
```

Reply:
Sure